### PR TITLE
Add authors and editors formatting

### PR DESCRIPTION
**Pull request by reinhardt**
Fills gaps in #43. Will also be used for citations.

syslabcom/scrum#903

### CODE REVIEW

File: src/recensio/plone/behaviors/editorial.py [hunk 1]
```diff
@@ -5,10 +5,13 @@
 from plone.dexterity.interfaces import IDexterityContent
 from plone.supermodel import model
 from recensio.plone import _
+from recensio.plone.utils import getFormatter
 from z3c.relationfield.schema import RelationChoice
 from z3c.relationfield.schema import RelationList
 from zope import schema
 from zope.component import adapter
+from zope.i18n import translate
```
Comment (ale-rt) [inline src/recensio/plone/behaviors/editorial.py]: Should we go for `plone.api.portal.translate`?

File: src/recensio/plone/behaviors/editorial.py [hunk 2]
```diff
@@ -66,3 +69,22 @@ def editorial(self):
     @editorial.setter
     def editorial(self, value):
         self.context.editorial = value
+
+    def get_formatted_editorial(self):
+        editors_list = [
+            getFormatter(" ")(editor.to_object.firstname, editor.to_object.lastname)
+            for editor in self.context.editorial
+        ]
```
Comment (ale-rt) [inline src/recensio/plone/behaviors/editorial.py]: This looks safer,  does not call twice `to_object` for every iteration and does initialize `getFormatter(" ")` only once:
```suggestion
        formatter = getFormatter(" ")
        editors_list = []
        for item in self.context.editorial:
            obj = item.to_object
            if obj:
                editors_list.append(formatter(obj.firstname, obj.lastname))
```

File: src/recensio/plone/content/review_article_collection.py [hunk 3]
```diff
@@ -48,8 +49,8 @@ class ReviewArticleCollection(Item):
     """Content-type class for IReviewArticleCollection."""
 
     def formatted_authors(self):
-        # TODO
-        return ""
+        authors_str = IAuthors(self).get_formatted_authors()
+        return authors_str
```
Comment (ale-rt) [inline src/recensio/plone/content/review_article_collection.py]: It looks weird to me that we have a methof on a behavior and a facade to it in the content type class.
Can we have it only in one place?

**Review by thet — APPROVED** [review]
:+1:

**Comment by thet** [comment]
aehm, wait a moment. I did something here - will restore the state!

**Review by ale-rt — COMMENTED** [review]
This is a partial review... the R was merged in the meanwhile...

**Pull request closed without merging**